Towny/src/main/java/com/palmergames/bukkit/towny/db/TownySQLSource.java

```diff
@@ -784,6 +784,12 @@ private boolean loadResident(Resident resident, ResultSet rs) {
 				plugin.getLogger().log(Level.WARNING, "Could not get lastOnline column on the residents table", e);
 			}
 			
+			try {
+				resident.setAbout(rs.getString("about"));
+			} catch (SQLException e) {
+				plugin.getLogger().log(Level.WARNING, "Could not get about column on the residents table");
+			}
+			
```

Comment:
```suggestion
			try {
				line = rs.getString("about");
				if (line != null)
					resident.setAbout(line);
			} catch (SQLException e) {
				plugin.getLogger().log(Level.WARNING, "Could not get about column on the residents table", e);
			}

```

This code will also have to get moved down below the line where `String line;` exists (line 835 on master branch.)